dev/make_boot_script

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env escript
+
+main(_) ->
+    {ok, Server} = reltool:start_server([
+        {config, "../rel/reltool.config"}
+    ]),
+    {ok, Release} = reltool:get_rel(Server, "couchdb"),
+    ok = file:write_file("devnode.rel", io_lib:format("~p.~n", [Release])),
+    ok = systools:make_script("devnode", [local]).
```

Comment:
This is really where the magic happens -- we take the existing `reltool.config` file and generate a .boot script with the `local` option, which automatically uses the existing paths for all the modules instead of relocatable ones like we have in the actual release.